core/spec.md

```diff
@@ -843,7 +855,7 @@ of the existing entity. Then the existing entity would be deleted.
   - MUST be a relative URL of the form `#JSON-POINTER` where the `JSON-POINTER`
     locates this entity within the current document. See [Doc Flag](#doc-flag)
     for more information.
-  - MUST NOT include `$details` suffix on its `<SINGULAR>id`.
+  - This URL MUST NOT include `$details` suffix after its `<SINGULAR>id`.
```

Comment:
Add an article here, e.g. "the `$details` suffix" or "a `$details` suffix"?

Reply:
done